lib/init/rc.boot

```diff
@@ -107,6 +107,10 @@ main() {
         }
     }
 
+    log "Loading rc.conf settings..."; {
+        [ -f /etc/rc.conf ] && . /etc/rc.conf
+    }
+
```

Comment:
Is `rc.conf` 100% limited to 'var=value'? I have a simpler alternative to `.` if so.